packages/zoe/test/unitTests/contracts/constantProduct/test-readmeScenario.js

```diff
@@ -0,0 +1,79 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava.js';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+import {
+  calcDeltaXSellingX,
+  calcDeltaYSellingX,
+  swapInNoFees,
+} from '../../../../src/contracts/constantProduct/core.js';
+import { makeRatio } from '../../../../src/contractSupport/index.js';
+import { pricesForStatedInput } from '../../../../src/contracts/constantProduct/calcSwapPrices.js';
+
+const BASIS_POINTS = 10000n;
+
+// Moola is central
+
+const setupMints = () => {
+  const moolaKit = makeIssuerKit('moola');
+  const bucksKit = makeIssuerKit('bucks');
+  const simoleanKit = makeIssuerKit('simolean');
+
+  const moola = value => AmountMath.make(moolaKit.brand, value);
+  const bucks = value => AmountMath.make(bucksKit.brand, value);
+  const simoleans = value => AmountMath.make(simoleanKit.brand, value);
+
+  return {
+    moolaKit,
+    bucksKit,
+    simoleanKit,
+    moola,
+    bucks,
+    simoleans,
+  };
+};
+
+// This test shows an example from README.md
```

Comment:
This test isn't new, is it? Was this supposed to be a file rename but the previous location didn't get deleted?

Reply:
The old test is being deleted from `.../test/unitTests/contracts/constantProduct/test-compareNewSwapPrice.js`.

The README example didn't have to change because newSwap and constantProduct AMM get the same answer.